pkg/controller/conditions.go

```diff
@@ -25,10 +25,44 @@ import (
 	v1 "kubevirt.io/client-go/api/v1"
 )
 
-type VirtualMachineConditionManager struct {
+type virtualMachineConditionManager struct {
 }
 
-func (d *VirtualMachineConditionManager) CheckFailure(vmi *v1.VirtualMachineInstance, syncErr error, reason string) (changed bool) {
```

Comment:
Why you changed this? it used to be exported and now it is not...

Reply:
Does the struct need to be exported? It has no use other than calling it's method, and it can be accessed through the exported method.

Reply:
If this struct is not used anywhere else, we can un-export it. But, since this is not related to this PR, maybe we can do it in a separate PR where we can discuss if it's required or not?

(I'm sorry for being picky on this...)